css/pages/navbar.css

```diff
@@ -91,6 +91,9 @@ nav .menu {
     font-size: 25px;
     line-height: 0;
   }
+  .login-btn{
+    top: -10px;
```

Comment:
Doesn't adding `bottom: 10px` work? Just wondering 🤔

Reply:
not sure why it did not work @swarajpure 
![image](https://user-images.githubusercontent.com/45538392/111598034-cd3c8e80-87f4-11eb-8441-ec9bc477b0bf.png)

Reply:
Can you please try margin/padding?

Reply:
![image](https://user-images.githubusercontent.com/45538392/111606300-142e8200-87fd-11eb-85a1-c2cfd962429d.png)
@swarajpure  please have a look